client/src/util/toasts/withToastProvider.js

```diff
@@ -7,22 +7,23 @@ import Toast from './Toast'
 function generateUEID () {
   let first = (Math.random() * 46656) | 0
   let second = (Math.random() * 46656) | 0
-  first = ('000' + first.toString(36)).slice(-3)
-  second = ('000' + second.toString(36)).slice(-3)
+  first = (`000${first.toString(36)}`).slice(-3)
+  second = (`000${second.toString(36)}`).slice(-3)
 
   return first + second
 }
 
 function withToastProvider (Component) {
   function WithToastProvider (props) {
     const [toasts, setToasts] = useState([])
-    const add = content => {
-      const id = generateUEID()
+    const providerValue = useMemo(() => ({
```

Comment:
Change this `useMemo` call to `useCallback`, since `useCallback(x, deps)` is equivalent to `useMemo(() => x, deps)` (see [React docs](https://reactjs.org/docs/hooks-reference.html#usecallback))